webapp/src/blocks.tsx

```diff
@@ -1744,9 +1744,7 @@ export class Editor extends toolboxeditor.ToolboxEditor {
                     let type = shadow.getAttribute('type');
                     const builtin = snippets.allBuiltinBlocks()[type];
                     let b = this.getBlockXml(builtin ? builtin : { name: type, attributes: { blockId: type } }, ignoregap, true);
-                    /* tslint:disable:no-inner-html setting one element's contents to the other */
```

Comment:
This one should also use that SDL rule

Reply:
> So is there a rule checking inner-html? imo we should keep that one around

Yes, there is a rule and these should all be brought back. I only realized there was a replacement rule (it wasn't found by the auto-migration tool) party way through the port so I missed some of these that I had already removed. I'll scan for all of these.

Reply:
Ah so this is one of the two files that crashes eslint today. After this PR I'm going to upgrade our TS version which I think will fix the crash, then it'll start linting and requiring directives here. For now, they wouldn't do anything.